.github/workflows/test.yml

```diff
@@ -28,13 +28,18 @@ jobs:
         rootless: ["rootless", ""]
         race: ["-race", ""]
         criu: ["", "criu-dev"]
+        dmz: ["", "runc_nodmz"]
```

Comment:
Do we really want offer the `runc_nodmz` option and expand the test matrix here? (even with the exlcude done afterwards)

Is there a downside of using the runc-dmz that people will want to opt-out of it?

Reply:
Cool. Too bad the fix can't work in all cases :(